tests/unit/test_notebooks.py

```diff
@@ -0,0 +1,61 @@
+import gzip
+import itertools
+import json
+import os
+import shutil
+from os.path import dirname, realpath
+
+TEST_PATH = dirname(dirname(realpath(__file__)))
+
+
+def test_optimize_criteo(tmpdir):
+    # datasets are stored zipped to avoid blowing up the repo, handle
+    dataset_path = os.path.join(TEST_PATH, "datasets", "criteo")
+    _unzip_dataset(dataset_path, tmpdir)
+
+    # read in the notebook
+    notebook_path = os.path.join(dirname(TEST_PATH), "examples", "optimize_criteo.ipynb")
+    os.environ["INPUT_PATH"] = str(tmpdir)
+    os.environ["OUTPUT_PATH"] = str(tmpdir)
+    source = _read_notebook_source(notebook_path)
+    exec(source, {})  # pylint: disable=exec-used
+
+
+def test_rossman_notebook(tmpdir):
+    # datasets are stored zipped to avoid blowing up the repo, handle
+    dataset_path = os.path.join(TEST_PATH, "datasets", "rossmann")
+    _unzip_dataset(dataset_path, tmpdir)
```

Comment:
why store any data in the repo. Why cant we just store on machine and mount the volume on the CI container?

Reply:
I've changed to generate random data rather than store in the repo